lib/metrics/events.js

```diff
@@ -53,9 +53,13 @@ const IGNORE_ROUTE_FLOW_EVENTS_FOR_PATHS = new Set([
   '/account/devices',
   '/account/profile',
   '/account/sessions',
-  '/certificate/sign'
+  '/certificate/sign',
+  '/password/forgot/status',
+  '/recovery_email/status'
```

Comment:
Do you care about either `/sms/status` or `/account/status`?

Reply:
Nope, on the basis that those aren't polled heavily like these two. It's hundreds-of-millions vs hundreds-of-thousands.